### ISSUE TITLE
[Overhaul] (Sec) Dominator - Energy Assault

### ISSUE THREAD

**Issue by Bopper-CoH** [Reborn-Team/Hero-Designer-DB#108]
- [x] Power Bolt
- [x] Bone Smasher
- [x] Power Push
- [x] Power Blast
- [x] Power Up
- [x] Whirling Hands
- [x] Total Focus
- [x] Sniper Blast
- [x] Power Burst

**Comment by Bopper-CoH** [comment]
Mostly complete. Power Up needs an overhaul still, and the EnergyFocus enum needs to be added. Once added, I can replace the ToHit97 enum placeholder with the EnergyFocus, which is why Power Burst and Bone Smasher show as unfinished.

**Comment by Bopper-CoH** [comment]
Completely finished. I noticed the Inherent of Energy Focus was already created, so I added it to Energy Assault. I gave Energy Focus the effect of granted 100% to GlobalChanceMod called "Energy". If we can add "EnergyFocus" as a GlobalChanceMod or Enum, we can change this mechanization, but for now it makes sense and it works (hopefully no other effects are flagged as Energy...which we should look into).

After adding this capability, i went back to fix Power Burst and Bone Smasher to work with Energy Focus. I also fixed the +Special and +Damage numbers for Power Up. The set is effectively complete. I did not add the chance for Energy Focus as it seems unnecessary and it requires the target to be stunned, which can't be done yet in Mids.
